velox/exec/fuzzer/RowNumberFuzzer.cpp

```diff
@@ -281,6 +283,7 @@ RowVectorPtr RowNumberFuzzer::execute(
   if (injectSpill) {
     spillDirectory = exec::test::TempDirectoryPath::create();
     builder.config(core::QueryConfig::kSpillEnabled, true)
+        .config(core::QueryConfig::kMaxSpillLevel, FLAGS_max_spill_level)
```

Comment:
Can you randomize this in test instead of using a flag?

Reply:
Got it, fixed.